packages/deployment/app/cloudformation.template.yml

```diff
@@ -708,7 +779,7 @@ Resources:
             - - "#cloud-config\n"
               - "\nruncmd:\n"
               - " - yum install -y aws-cfn-bootstrap\n"
-              - Fn::Sub: " - /opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource CumulusContainerInstanceLaunch --region ${AWS::Region}\n"
+              - Fn::Sub: " - /opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource CumulusContainerInstanceLaunch --region ${AWS::Region} -c MountConfig\n"
```

Comment:
what is implication of this change if there is no efs volume?

Reply:
no implication.  see related comment above.